src/test/unit/engine/NetworkFactoryTest.cpp

```diff
@@ -30,15 +30,23 @@
 #include <nupic/engine/Network.hpp>
 #include <nupic/engine/NuPIC.hpp>
 #include <nupic/engine/Region.hpp>
+#include <nupic/os/Path.hpp>
 #include <nupic/utils/Log.hpp>
 
 using namespace nupic;
 
+// macro to ease nesting
+#define GP Path::getParent
+
+const std::string UNIT_TESTS_EXECUTABLE_PATH = Path::getExecutablePath();
+const std::string NUPIC_CORE_PATH =  GP(GP(GP(GP(UNIT_TESTS_EXECUTABLE_PATH))));
+const std::string PATH_TO_FIXTURES = Path::join(NUPIC_CORE_PATH, "/src/test/unit/engine/fixtures");
```

Comment:
These path separators "/" will probably not work well on Windows. If you look at the implementation of the various `Path::join` overloads, it appears that they weren't intended to be used with path segments that already contain path separators, but are designed to add path separators to path segments.

CC @scottpurdy @andrewmalta13